### PR TITLE
A bunch of minor improvements and corrections

### PR DESCRIPTION
**Pull request by mjahoda**

### CODE REVIEW

File: modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ Anchor names:: Provide an anchor in the format `+++[id='anchor-name_{context}']+
 [id='anchor-name_{context}']
 = Module Heading
 
-First sentence of topic.
+The first sentence of topic.
```
Comment (jenmalloy) [inline modular-docs-manual/content/topics/module_anchor-and-file-names-concept.adoc]: of the topic

File: modular-docs-manual/content/topics/module_guidelines-concept.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@ A concept module requires a short introduction and optionally, can also include
 
 When planning a concept module, look at nouns in related procedure modules and assemblies to find the concepts to explain to users.
 Explain only things that are visible to users.
-Even if a concept is interesting, it probably does not require explanation if it is not visible to users.
+Even if a concept is interesting, it probably does not require an explanation if it is not visible to users.
```
Comment (jenmalloy) [inline modular-docs-manual/content/topics/module_guidelines-concept.adoc]: "an" is not necessary here, so in the interest of minimalism (use as few words as possible), I would not add it

**Comment by adahms** [comment]
@mjahoda - Thank you for the contribution!

Based on the above, it looks like there are two minor points that might need to be addressed before this pull request can be accepted.

Would you be able to take a look and update the PR accordingly?

File: modular-docs-manual/content/topics/module_guidelines-concept.adoc (hunk 2 above)
Comment (mjahoda) [reply modular-docs-manual/content/topics/module_guidelines-concept.adoc]: @jenmalloy Thank you for the feedback - I've added the missing article (above). However, I still believe that "an" fits here. I'm not a native speaker though.

**Comment by mjahoda** [comment]
> Would you be able to take a look and update the PR accordingly?

@adahms I've updated the PR but I haven't incorporated the second comment by @jenmalloy. Please, feel free to delete the indefinite article before "explanation" if you also believe that it's better without it.

**Comment by adahms** [comment]
@mjahoda - Thanks for your reply, and sorry to keep you waiting!

I can see an argument in either direction, but the sentence is grammatically correct and conveys the intended meaning in its current format. Given that the change being discussed is a minor one - and that this pull request has been open for some time now - I am inclined to accept it so that we can incorporate the work you have done.

**Comment by adahms** [comment]
LGTM; merging.

All - please let me know if you feel additional work is required for this pull request and we'll investigate it in a separate branch.

**Pull request merged**